modules/core/05-port/types/query.pb.go

```diff
@@ -8,7 +8,7 @@ import (
 	fmt "fmt"
 	grpc1 "github.com/gogo/protobuf/grpc"
 	proto "github.com/gogo/protobuf/proto"
-	types "github.com/line/ibc-go/v3/modules/core/04-channel/types"
+	types "github.com/Finschia/ibc-go/v3/modules/core/04-channel/types"
```

Comment:
I'm not sure that it isn't effective for the binary section. Would you check it?

Reply:
I believe this binary is determined by how the proto file is modified. But I don't know because there is no proto file in the first place.

https://github.com/Finschia/ibc-go/blob/772269e26a633a208c9bbe3b67f36ddf07804519/modules/core/05-port/module.go#L21-L24
I found the top caller that call this file but the function is not used anywhere.

https://github.com/cosmos/ibc-go/pull/2045/files
And, I found the pr that fix the same issue in the original repo. May I merge this pr?

Reply:
I would like to backport https://github.com/cosmos/ibc-go/pull/2045 if I were.

Reply:
@ulbqb , if then, could you fix this issue as another PR?

Reply:
I'll fix this in another PR.